matlab_simulation/code v1.0/ME597-8-NLP/Trajectory_bazier.m

```diff
@@ -1,5 +1,20 @@
+%% This file develops a desired trajectory using Bazier Curves.
+%% P is a set of points including start point, bazier control point and target.
+
 function xdT=Trajectory_bazier(xd_start,xd_end,dt,TTot)
 xd_midpoint=(xd_end(1)-xd_start(1)).*(rand(1,2));
 P=[xd_start;xd_start+xd_midpoint;xd_end];
-BezierCurve(P); %generates Bz_fun
+
+syms g
+Np = size(P, 1); 
+

```

Comment:
What is all this? Explain what you are doing in comments.  Add references or links to wikipedia if not covered in the course notes.